core/src/dird/dird_conf.cc

```diff
@@ -255,6 +255,8 @@ static ResourceItem cli_items[] = {
   { "NdmpLogLevel", CFG_TYPE_PINT32, ITEM(res_client, ndmp_loglevel), 0, CFG_ITEM_DEFAULT, "4", NULL, NULL },
   { "NdmpBlockSize", CFG_TYPE_SIZE32, ITEM(res_client, ndmp_blocksize), 0, CFG_ITEM_DEFAULT, "64512", NULL, NULL },
   { "NdmpUseLmdb", CFG_TYPE_BOOL, ITEM(res_client, ndmp_use_lmdb), 0, CFG_ITEM_DEFAULT, "true", NULL, NULL },
+  { "TlsDowngrade", CFG_TYPE_BOOL, ITEM(res_client, tls_downgrade), 0, CFG_ITEM_DEFAULT, "false", "23.0.0",
```

Comment:
I'm not sure that I'm happy with that naming.
Maybe it is worth investigating how `TLS Require = yes` would behave in that  context. I think setting `TLS Require = yes` should not allow a downgrade to cleartext.